docs/adr-0001-publishing_metric_docs.md

```diff
@@ -0,0 +1,139 @@
+* …
+
+### Negative Consequences
+
+* [e.g., compromising quality attribute, follow-up decisions required, …]
+* …
+
+-->
+
+## Pros and Cons of the Options <!-- optional -->
+
+### "metadata.json": Save some JSON to GCS
+
+Imagine a file named something like `metadata_<experiment_slug>.json` containing objects that look like:
+
+```json
+{
+    "metrics": {
+        "active_hours": {
+            "friendly_name": "Active hours",
+            "description": "Measures the amount of time (in 5-second increments) during which Firefox received user input from a keyboard or mouse. The Firefox window does not need to be focused.",
+            "bigger_is_better": true
+        },
+        "uri_count": {
+            "friendly_name": "URIs visited",
+            "description": "Counts the total number of URIs visited. Includes within-page navigation events (e.g. to anchors).",
+            "bigger_is_better": true
+        },
+        "ever_used_picture_in_picture": {
+            "friendly_name": "Ever used PiP",
+            "description": "Whether each client ever used Picture in Picture during the measurement window.",
+            "bigger_is_better": true
+        },
+        "picture_in_picture_count": {
+            "friendly_name": "PiP use count",
+            "description": "How many times each client used Picture in Picture during the measurement window.",
+            "bigger_is_better": true
+        }
+    },
+    "probesets": {
+        "picture_in_picture": [
+            "ever_used_picture_in_picture",
+            "uses_of_picture_in_picture"
+        ]
+    }
+}
+```
+
+Each member of the `probesets` object is an ordered list.
+The first metric in the list is assumed to be the most interesting metric to display.
+It will typically be a conversion metric.
+
+We'll put it in the mozanalysis GCS bucket next to the json files containing the statistical summaries for the experiment.
+
+* +: One-stop shopping; Jetstream's consumers are already looking for data in this GCS bucket.
+* +: Seems easy and complete
+
+### A single metadata database
+
+Many metrics appear identically in more than one experiment.
+Instead of producing a separate file for each experiment,
+we could produce a shared database that describes all experiments.
+
+This has some disadvantages associated with the need to ensure that this information is consistent:
+
+* Metrics can be changed or removed from mozanalysis --
+  if the dashboard presents a definition that was updated since the experiment was analyzed,
+  it could be incorrect, which I think is worse than making it hard to fix typos or stale information.
+* Regenerating the list of all known metrics requires comprehending all experiment configurations that have ever been processed;
+  without some extra work, it also implies a new requirement that experiment configurations should never use a metric name
+  that has been used before with a different definition, which is too constraining.
+
```

Comment:
Also requires someone maintaining the database, if we would go for an actual database instead of just a file.